src/server-dev.ts

```diff
@@ -58,7 +58,7 @@ process.nextTick(async () => {
                         filterExistingFlagNames: true,
                         teamsIntegrationChangeRequests: true,
                         simplifyDisableFeature: true,
-                        adminNavUI: true,
+                        adminNavUI: false,
```

Comment:
I wonder why we're turning this off?

Reply:
As @thomasheartman comments further down - currently there's no menu-way of getting back out. So I didn't want it to break life for us during dev, but I also want to break this up into reviewable PRs